rectools/models/nn/transformers/base.py

```diff
@@ -601,20 +602,34 @@ def __setstate__(self, state: tp.Dict[str, tp.Any]) -> None:
         self.__dict__.update(loaded.__dict__)
 
     @classmethod
-    def load_from_checkpoint(cls, checkpoint_path: tp.Union[str, Path]) -> tpe.Self:
-        """
-        Load model from Lightning checkpoint path.
+    def load_from_checkpoint(
+        cls,
+        checkpoint_path: tp.Union[str, Path],
+        map_location: tp.Union[str, torch.device, None] = None,
+        config_update: tp.Dict[str, tp.Any] = {},
+    ) -> tpe.Self:
+        """Load model from Lightning checkpoint path.
 
         Parameters
         ----------
         checkpoint_path: Union[str, Path]
             Path to checkpoint location.
+        map_location: Union[str, torch.device, None], default None
+            Target device to load the checkpoint (e.g., 'cpu', 'cuda:0').
+            If None, will use the device the checkpoint was saved on.
+        config_update: tp.Dict[str, tp.Any], default '{}'
+            Contains custom values for checkpoint['hyper_parameters'].
+            Config_update has to be flattened with 'dot' reducer, before passed.
```

Comment:
Since we are expecting a flat dict for this config, let's call it `params` to keep naming equal throughout our code everywhere.
Let's call it `model_params_update`.
We just call this form "a flat dict of model hyper-parameters" or "a model config in flatten form"

Let's also explicitly write here that "You can use this argument to remove training-specific parameters that are not needed anymore. e.g. `get_trainer_func`"